Documented Channel Info functionality (#6097)
mattermost/docs: source/channels/set-channel-preferences.rst

.. image:: ../images/channel-header.png
    :alt: Channel headers can include links to documents, tools, or websites.

.. tip::

>   Alternatively, you can provide or update a channel header by selecting the channel name, then selecting **Edit Channel Header** from the list of options.

Reviewer: ```suggestion
  Alternatively, to provide or update a channel header, select the channel name, then select **Edit Channel Header** from the list of options.
```

Channel purpose
---------------

A channel purpose refers to text that displays when users select **View Info** for a channel. A channel purpose can be up to 250 characters in length and is often used to help users decide whether to join the channel.

> Provide a channel purpose by selecting **Edit Channel Purpose**. Any channel member can change a channel purpose, unless the System Admin has `restricted permissions to do so using advanced permissions </onboard/advanced-permissions.html>`__.

Reviewer: ```suggestion
To add a channel purpose, select **Edit Channel Purpose**. Any channel member can change a channel purpose, unless the System Admin has `restricted permissions to do so using advanced permissions </onboard/advanced-permissions.html>`__.
```

.. image:: ../images/channel-purpose.png
    :alt: Channel purpose helps users decide if they want to join the channel based on its scope or focus.

.. tip::

>   Alternatively, you can provide or update a channel purpose by selecting the channel name, then selecting **Edit Channel Purpose** from the list of options.

Reviewer: ```suggestion
  Alternatively, to provide or update a channel purpose, select the channel name, then select **Edit Channel Purpose** from the list of options.
```

Channel name
------------

A channel name must be at least two characters, and can be up to 64 characters in length. `Some unicode characters <https://www.w3.org/TR/unicode-xml/#Charlist>`_ aren't supported.

> Change the channel name by selecting **Rename Channel**. Changing the name of the channel also changes the channel URL. Any channel member can change a channel name, unless the System Admin has `restricted permissions to do so using advanced permissions </onboard/advanced-permissions.html>`__.

Reviewer: ```suggestion
To change the channel name, select **Rename Channel**. Changing the name of the channel also changes the channel URL. Any channel member can change a channel name, unless the System Admin has `restricted permissions to do so using advanced permissions </onboard/advanced-permissions.html>`__.
```
